stanford_profile.install

```diff
@@ -30,3 +30,22 @@ function stanford_profile_update_9105() {
     \Drupal::database()->truncate($table)->execute();
   }
 }
+
+/**
+ * Removes content from taxonomy terms.
+ */
+function stanford_profile_update_9106() {
+  $vocabulary_names = [
+    'opportunity_tag_filters',
+    'opportunity_type',
+    'opportunity_sponsor'
+  ];
+
+  foreach ($vocabulary_names as $vocabulary_name) {
+    $terms = \Drupal::entityTypeManager()->getStorage('taxonomy_term')->loadTree($vocabulary_name, 0, NULL, TRUE);
+
+    foreach ($terms as $term) {
+      $term->delete();
+    }
+  }
```

Comment:
```suggestion
    $terms = \Drupal::entityTypeManager()->getStorage('taxonomy_term')->loadByProperties(['vid' => $vocabulary_name]);

    foreach ($terms as $term) {
      $term->delete();
  }
```

Reply:
@pookmish Don't I still need the foreach loop, to provide the vocabulary name?

Reply:
@pookmish Never mind. I see it takes an array.